packages/Webkul/Installer/src/Console/Commands/Installer.php

```diff
@@ -175,13 +177,13 @@ protected function askForApplicationDetails()
             env('APP_URL', 'http://localhost:8000')
         );
 
-        $this->envUpdate(
+        $timezones = $this->getTimezones();
+        $defaultTimezone = $this->updateEnvSuggest(
             'APP_TIMEZONE',
-            date_default_timezone_get()
+            'Please select the application timezone',
+            $timezones
```

Comment:
We can manage this action using the existing `updateEnvChoice` method. Adding a new method for a single action is unnecessary.